.github/workflows/scripts/post_before_script.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+
+echo "machine pulp
+login admin
+password password
+" > ~/.netrc
+
+chmod og-rw ~/.netrc
```

Comment:
This file was not ported from travis - https://github.com/pulp/pulp_ansible/blob/master/.travis/post_before_script.sh

Reply:
only pulp-ansible required this chmod:
https://github.com/pulp/pulp_ansible/pull/315#discussion_r427820658